src/segment_kubectl.go

```diff
@@ -31,15 +51,80 @@ func (k *kubectl) init(props *properties, env environmentInfo) {
 }
 
 func (k *kubectl) enabled() bool {
+	parseKubeConfig := k.props.getBool(ParseKubeConfig, false)
+	if parseKubeConfig {
+		return k.doParseKubeConfig()
+	} else {
+		return k.doCallKubectl()
+	}
+}
+
+func (k *kubectl) doParseKubeConfig() bool {
+	// Follow kubectl search rules (see https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#the-kubeconfig-environment-variable)
+	// TL;DR: KUBECONFIG can contain a list of files. If it's empty ~/.kube/config is used. First file in list wins when merging keys.
+	kubeconfigs := filepath.SplitList(k.env.getenv("KUBECONFIG"))
+	if len(kubeconfigs) == 0 {
+		kubeconfigs = []string{filepath.Join(k.env.homeDir(), ".kube/config")}
+	}
+	contexts := make(map[string]KubeConfigContext)
+	k.Context = ""
+	for _, kubeconfig := range kubeconfigs {
+		if kubeconfig == "" {
```

Comment:
I always default to using `len(kubeconfig) == 0`

Reply:
Personally I prefer `==""` as it's clear I'm checking a string and not an array or anything else. But when in Rome... ;) I replaced all occurances of `== ""` with the preferred pattern.